sci-visualization/gnuplot/gnuplot-5.2.8.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Command-line driven graphing utility"
+DESCRIPTION="Originally created to allow both students and scientists to \
+visualize mathematical functions and data interactively, but has grown to \
+support many non-interactive uses such as web scripting. It is also used \
+as a plotting engine by third-party applications like Octave."
+HOMEPAGE="http://www.gnuplot.info/"
+COPYRIGHT="1986-1993, 1998, 2004-2019 Thomas Williams, Colin Kelley"
+LICENSE="Gnuplot"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/gnuplot/gnuplot-$portVersion.tar.gz"
+CHECKSUM_SHA256="60a6764ccf404a1668c140f11cc1f699290ab70daa1151bb58fed6139a28ac37"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	gnuplot$secondaryArchSuffix = $portVersion
+	cmd:gnuplot = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgd$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libtiff$secondaryArchSuffix
+	lib:libvpx$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcairo$secondaryArchSuffix
+#	devel:libcerf$secondaryArchSuffix
+	devel:libgd$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+#	devel:libpango_1.0$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix >= 8
+	devel:libvpx$secondaryArchSuffix
```

Comment:
This should (and other PR's atm) be >= 6
```diff
- 	devel:libvpx$secondaryArchSuffix
+ 	devel:libvpx$secondaryArchSuffix >= 6
```

Reply:
No, it should not; this will be done automatically.